sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -418,6 +436,39 @@ abstract class SessionCatalogSuite extends AnalysisTest {
     }
   }
 
+  test("rename global temp table") {
+    withBasicCatalog { catalog =>
+      val globalTempTable = Range(1, 10, 2, 10)
+      catalog.createGlobalTempView("tbl1", globalTempTable, overrideIfExists = false)
+      assert(catalog.getGlobalTempView("tbl1") == Option(globalTempTable))
+      assert(catalog.externalCatalog.listTables("db2").toSet == Set("tbl1", "tbl2"))
+      // If database is not specified, global temp table will not be renamed
```

Comment:
`global temp table` -> `global temp views`. This is a general comment. We do not have `global temp table`